src/app/public/modules/radio/radio.component.ts

```diff
@@ -216,6 +216,14 @@ export class SkyRadioComponent implements OnDestroy, ControlValueAccessor {
     }
   }
 
+  /**
+   * Fires when users focus off a radio button.
+   */
+  @Output()
+  public get blur(): Observable<SkyRadioChange> {
```

Comment:
To avoid adding to the public API, we could just make this a public property on the component (unsure about the `boolean` value or what the radio group needs).
```
/**
 * @internal
 */
public blur = new BehaviorSubject<boolean>(false);
```